static/js/topic_page/topic_page.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from "react";
+import ReactDOM from "react-dom";
+
+import { MainPane } from "./main_pane";
+
+window.onload = () => {
+  renderPage();
+};
+
+function renderPage(): void {
+  const urlParams = new URLSearchParams(window.location.search);
+  // Get topic and render menu.
+  const topic = urlParams.get("topic");
```

Comment:
as we pass the config proto to the client, we can read the topic from there too.